### PR TITLE
Chore/configure GitHub actions

### PR DESCRIPTION
**Pull request by AbdulmajeedAlOmari**

### CODE REVIEW

File: ClassiVault.Api/Controllers/VaultsController.cs
Comment (AhmedAlzubaidi) [inline ClassiVault.Api/Controllers/VaultsController.cs:1]: change passwordVault to vault.

File: ClassiVault.Tests/VaultServiceTests.cs
Comment (AhmedAlzubaidi) [inline ClassiVault.Tests/VaultServiceTests.cs:1]: change passwordVault to vault.

File: Services/IPasswordVaultService.cs
Comment (AhmedAlzubaidi) [inline Services/IPasswordVaultService.cs:1]: change IPasswordVaultService to IVaultService.

**Review by AhmedAlzubaidi — CHANGES_REQUESTED** [review]

File: ClassiVault.Api/Controllers/VaultsController.cs
Comment (AbdulmajeedAlOmari) [reply ClassiVault.Api/Controllers/VaultsController.cs:1]: Where is passwordVault?

File: ClassiVault.Tests/VaultServiceTests.cs
Comment (AbdulmajeedAlOmari) [reply ClassiVault.Tests/VaultServiceTests.cs:1]: where is passwordVault?

File: Services/IPasswordVaultService.cs
Comment (AbdulmajeedAlOmari) [reply Services/IPasswordVaultService.cs:1]: This is an extra file that was merged in a previous MR by mistake, I just removed it.

File: ClassiVault.Api/Controllers/VaultsController.cs
Comment (AbdulmajeedAlOmari) [reply ClassiVault.Api/Controllers/VaultsController.cs:1]: Done

File: ClassiVault.Tests/VaultServiceTests.cs
Comment (AbdulmajeedAlOmari) [reply ClassiVault.Tests/VaultServiceTests.cs:1]: Done

**Review by AhmedAlzubaidi — APPROVED** [review]

**Pull request merged**